packages/core/src/app/common/error/SentryErrorLogger.ts

```diff
@@ -1,4 +1,4 @@
-import { captureException, init, withScope, BrowserOptions, Event, Integrations, Severity, StackFrame } from '@sentry/browser';
+import { captureException, init, withScope, BrowserOptions, Event, Integrations, SeverityLevel, StackFrame } from '@sentry/browser';
```

Comment:
`Severity` is deprecated. 
https://github.com/getsentry/sentry-javascript/blob/master/packages/types/src/severity.ts